scripts/ci/goreleaser-nightly.yml

```diff
@@ -0,0 +1,174 @@
+project_name: influxdb2
```

Comment:
Instead of creating a near-copy of the goreleaser config, can we use env variables to inject different values for the pieces of this that we want to change between builds? The templating system allows for use of things like `{{.Env.FOO_VARIABLE}}`

Reply:
That works for appending the `-nightly` suffix to the version, but I'm not sure how to manage various combinations of s3 buckets for artifact uploads with a single goreleaser file. I'll think about it some more.